docs/docs/launch-manual/sections/core-hacking/sections/building-pulsar.md

```diff
@@ -98,15 +102,13 @@ git submodule init && git submodule update
 Now install and build Pulsar & ppm:
 
 ```sh
-yarn install
-yarn build
-yarn build:apm
+yarn install && yarn build && yarn build:apm
 ```
 
 Start Pulsar!
 
 ```sh
-yarn start
+yarn start --no-sandbox
```

Comment:
I'm torn about this one still. This isn't OpenSUSE specific and really this needs to be decided at a more global level as this is relevant to every linux release depending on graphics card etc. If this is set internally then we don't need this at all.

Reply:
This is actually under the heading `### Building and running the application` I believe.

> needs to be decided at a more global level

Quite possibly, yeah. I know there is some discussion to add `--no-sandbox` to the `pulsar.sh`. I'm not familiar off the top of my head whether that is called when you do `yarn start` or if it hits the Electron binary (renamed/rebranded to "pulsar") directly? if `yarn start` skips `pulsar.sh` then it might actually be valid to add this flag here in these docs.

Reply:
Well one thing I was thinking when I said global was if pulsar.sh gets used in appimage or the tarball. If not then should we not have some other mechanism to accomplish its tasks?

As for 'yarn start' this would be better to add to the package script rather than every instance of the docs.

Reply:
On my machine it just wouldn't start if I didn't put the no sandbox flag

Reply:
Yes but the point is that it if we need it to always be in the instructions then it should be in the script like I mention above so we don't need to keep editing the docs. I'll probably make a PR to see if we want to add that.

Reply:
I've made a PR for this, depending on the outcome of it will depend if we decide to add `--no-sandbox` to the instructions.
https://github.com/pulsar-edit/pulsar/pull/276